include/cose/cose_configure.h.in

```diff
@@ -1,20 +1,39 @@
 //
-//  Determine which cryptographic library we are going to be using
+// cose_configure.h. Generated from cose_configure.h.in by CMake
+// Determine which cryptographic library we are going to be using
 //
 
 #pragma once
 
-#if defined(USE_MBED_TLS)
-#if defined(USE_OPEN_SSL) || defined(USE_BCRYPT)
```

Comment:
@jimsch `USE_BCRYPT` what about this? is this still supported?

Reply:
I stopped working on the task of allowing for BCRYPT to be used.  It was never supported due to that.  Let me think about the question of removing it.

Reply:
does not have to be in this MR. lets create an issue for that